src/Integration/Helpers/ToolWIndowService.cs

```diff
@@ -40,15 +40,15 @@ public ToolWindowService([Import(typeof(SVsServiceProvider))] IServiceProvider s
             Debug.Assert(shell != null, "Failed to retrieve the IVsUIShell");
         }
 
-        public void Show(Guid toolWindowId)
+        public void Show(Guid toolWindowId, bool activateWindow)
         {
             // We want VS to ask the package to create the tool window if it doesn't already exist
             const uint flags = (uint)__VSFINDTOOLWIN.FTW_fForceCreate;
 
             var hr = shell.FindToolWindow(flags, toolWindowId, out var windowFrame);
             Debug.Assert(ErrorHandler.Succeeded(hr), $"Failed to find tool window. Guid: {toolWindowId}, hr: {hr} ");
 
-            if (ErrorHandler.Succeeded(hr))
+            if (activateWindow && ErrorHandler.Succeeded(hr))
```

Comment:
This is the actual product change.
All of the other product code changes are to pass `true` as the additional parameter.